app.py

```diff
@@ -29,22 +31,17 @@
 # Enable Flask-Migrate commands "flask db init/migrate/upgrade" to work
 migrate = Migrate(app, db)
 
-# Create databases, if databases exists doesn't issue create
-# For schema changes, run "flask db migrate"
+# The import must be done after db initialization due to circular import issue
 from models import Restaurant, Review
-db.create_all()
-db.session.commit()
 
 @app.route('/', methods=['GET'])
 def index():
-    from models import Restaurant
```

Comment:
These imports should not be needed inside the route functions since these files were already imported earlier. Everything I've tried in the app works correctly without them.